FileAPI/snapshot-state/touch-manual.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<label>Select a file first: <input type="file" id="fileInput"></label><br>
+<button id="readyButton">Then touch the selected file and click this</button>
```

Comment:
Nit: Maybe write "Then update the last modified timestamp ('touch') the selected file..."

Reply:
```suggestion
<button id="readyButton">Then update the last modified timestamp ('touch') of the selected file and click this</button>
```